standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/utils/MetaStoreUtils.java

```diff
@@ -244,6 +244,13 @@ public static boolean isExternalTable(Table table) {
     return isExternal(params);
   }
 
+  public static boolean isTranslatedToExternalTable(Table table) {
+    Map<String, String> p = table.getParameters();
```

Comment:
Can you change the name of this variable 'p' to something more meaningful like tblProperties?